lib/util/fairplay_utils.js

```diff
@@ -150,6 +150,20 @@ shaka.util.FairPlayUtils = class {
     request.body = shaka.util.StringUtils.toUTF8('spc=' + base64Payload);
   }
 
+  /**
+   * EZDRM FairPlay request.
+   *
+   * @param {shaka.net.NetworkingEngine.RequestType} type
+   * @param {shaka.extern.Request} request
+   * @export
+   */
+  static ezdrmFairPlayRequest(type, request) {
+    if (type !== shaka.net.NetworkingEngine.RequestType.LICENSE) {
+      return;
+    }
+    request.headers['Content-Type'] = 'application/octet-stream';
```

Comment:
Is this not the default?  Or do we leave off `Content-Type` by default (and leave it to servers to assume "application/octet-stream")?

Reply:
By default we leave `Content-Type` headers off, except for some PlayReady license requests.